test/torch/tensors/test_autograd.py

```diff
@@ -511,3 +511,63 @@ def test_get_float_prec_on_autograd_tensor(workers):
     x2 = x.fix_precision()
     x2 = x2.share(bob, alice, crypto_provider=james, requires_grad=True)
     assert (x2.get().float_precision() == x).all()
+
+
+def test_serialize_deserialize_autograd_tensor(workers):
+    # let's try to send an autogradTensor to a remote location and get it back
+    alice = workers["alice"]
+    random_tensor = torch.randn(5, 3)
+    remote_tensor = random_tensor.send(alice, local_autograd=True)
+    local_tensor = remote_tensor.get()
+    # check if the tensor sent is equal to the tensor got from the remote version
+    equal_tensors = torch.all(torch.eq(local_tensor, random_tensor))
+
+    assert equal_tensors == 1
+
+
+def test_train_remote_autograd_tensor(workers):
+    def train(model_input, data_input, target_input):
+        opt = optim.SGD(params=model_input.parameters(), lr=0.1)
+        for iter in range(10):
+            # 1) erase previous gradients (if they exist)
+            opt.zero_grad()
+            # 2) make a prediction
+            predictions = model_input(data_input)
+            # 3) calculate how much we missed
+            loss = ((predictions - target_input) ** 2).sum()
+            # 4) Figure out which weights caused us to miss
+            loss.backward()
+            # 5) change those weights
+            opt.step()
+        return (loss, model_input)
+
+    # Training procedure to train an input model, be it remote or local
```

Comment:
The comment ( # Training procedure) should be attached to the nested train() function

Reply:
Yeah, I'll fix it.